docs/js.md

```diff
@@ -0,0 +1,162 @@
+```
+
+### [Built in Values: NaN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Infinity)
+
+*_symbol_: 'NaN'
+*_pronunciation_: "not a number", "nahn"
+*_examples_:
+```javascript
+"cat" == NaN
+```
+
+### [Built in Values: Undefined](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Infinity)
+
+*_symbol_: 'undefined'
+*_pronunciation_: "un-defined"
+*_examples_:
+```javascript
+
+```
+
+
+### [Operator: Addition](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Arithmetic_Operators#Addition_(.2B))
+
+* _symbol_: `+`
+* _pronunciation_: "plus", "added to"
+* _examples_:
+```javascript
+1 + 2 + 3 // 6, of course
+'1' + 2 + 3 // '123', obviously... WTF?
+```
+
+### [Operator: Modulus](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Expressions_and_Operators#Arithmetic_operators)
+
+*_symbol_: `%`
+*_pronunciation_: "mod-u-lus"
+*_example_:
+```javascript
+43 % 10
+Returns 3
+```
+
+### [Operator: Increment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Expressions_and_Operators#Arithmetic_operators)
+
+*_symbol_: '++'
+*_pronunciation_: "increment"
+*_example_:
+```javascript
+12++ == 13
+```
+
+### [Operator: Decrement](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Expressions_and_Operators#Arithmetic_operators)
+
+*_symbol_: '--'
+*_pronunciation_: "decrement"
+*_example_:
+```javascript
+12-- == 11
+```
+
+### [Operator: Unary Negation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Expressions_and_Operators#Arithmetic_operators)
+
+*_symbol_: `-`
+*_pronunciation_: "unary negation"
+*_example_:
+```javascript
+If x is 10, then -x returns -10
+```
+
+### [Operator: Assignment =+](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Expressions_and_Operators#Assignment_operators)
+
+*_symbol_: '+='
+*_pronunciation_: "equals x plus"
+*_example_:
+```javascript
+x += y == x = x + y
+```
+
+### [Operator: Assignment -=](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Expressions_and_Operators#Assignment_operators)
+
+*_symbol_: '-='
+*_pronunciation_: "equal x minus"
+*_example_:
+```javascript
+x -= y == x = x - y
+```
+
+### [Operator: Assignment *=](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Expressions_and_Operators#Assignment_operators)
+
+*_symbol_: '*='
+*_pronunciation_: "equals x times"
+*_example_:
+```javascript
+x *= y == x = x * y
+```
+
+### [Operator: Assignment /=](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Expressions_and_Operators#Assignment_operators)
+
+*_symbol_: '/='
+*_pronunciation_: "equals x divided"
+*_example_:
+```javascript
+x /= y == x = x / y
+```
+
+### [Operator: Assignment %=](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Expressions_and_Operators#Assignment_operators)
+
+*_symbol_: '%='
+*_pronunciation_: "equal x modulus"
+*_example_:
+```javascript
+x %= y == x = x % y 
+```
```

Comment:
Hey, good guess! Asking a question like this on [Stack Overflow](http://stackoverflow.com) (or just searching for it first, can get you trolled but also might get you a decent answer.